internal/rest/avi_obj_vsvip.go

```diff
@@ -201,6 +204,9 @@ func (rest *RestOperations) AviVsVipBuild(vsvip_meta *nodes.AviVSVIPNode, vsCach
 						}
 					}
 				}
+				if vipNetwork.V6Cidr != "" {
```

Comment:
Line number 187, 188: will be true if only v6 given. I think we should not print warning if v6cidr is given.